app/models/waste_exemptions_engine/feature_toggle.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require "yaml"
+
+module WasteExemptionsEngine
+  class FeatureToggle
+    def self.active?(feature_name)
+      feature_toggles[feature_name] && feature_toggles[feature_name][:active]
+    end
+
+    class << self
+      private
+
+      # rubocop:disable Style/ClassVars
+      def feature_toggles
+        @@feature_toggles ||= load_feature_toggles
+      end
+      # rubocop:enable Style/ClassVars
+
+      def load_feature_toggles
+        HashWithIndifferentAccess.new(YAML.load_file(file_path))
+      end
+
+      def file_path
+        Rails.root.join("config/feature_toggles.yaml")
```

Comment:
This may be the nitpickiest thing ever but we usually use a .yml suffix instead of .yaml. So maybe worth a switch for consistency's sake.

Reply:
ooh, good spot! 🔍 🕵️‍♀️

Reply:
Ah! Nice one! Will change it!